refinery/analysis_manager/schemas/AnalysisConfig.json

```diff
@@ -0,0 +1,92 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema",
+  "oneOf": [
+    {
+      "required": [
+        "custom_name",
+        "studyUuid",
+        "toolUuid",
+        "user_id",
+        "workflowUuid"
+      ],
+      "properties": {
+        "custom_name": {
+          "type": "string"
+        },
+        "studyUuid": {
+          "type": "string",
+          "pattern": "^[0-9a-f]{8}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{12}$"
+        },
+        "toolUuid": {
+          "type": "string",
+          "pattern": "^[0-9a-f]{8}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{12}$"
+        },
+        "user_id": {
+          "type": "number"
+        },
+        "workflowUuid": {
+          "type": "string",
+          "pattern": "^[0-9a-f]{8}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{12}$"
+        }
+      }
+    },
+    {
+      "required": [
+        "custom_name",
+        "nodeSetUuid",
+        "studyUuid",
+        "user_id",
+        "workflowUuid"
+      ],
+      "properties": {
+        "custom_name": {
+          "type": "string"
+        },
+        "nodeSetUuid": {
+          "type": "string",
+          "pattern": "^[0-9a-f]{8}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{12}$"
+        },
+        "studyUuid": {
+          "type": "string",
+          "pattern": "^[0-9a-f]{8}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{12}$"
+        },
+        "user_id": {
+          "type": "number"
+        },
+        "workflowUuid": {
+          "type": "string",
+          "pattern": "^[0-9a-f]{8}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{12}$"
+        }
+      }
+    },
+    {
+      "required": [
+        "custom_name",
+        "nodeRelationshipUuid",
+        "studyUuid",
+        "user_id",
+        "workflowUuid"
+      ],
+      "properties": {
+         "custom_name": {
+          "type": "string"
+        },
+        "nodeSetUuid": {
+          "type": "string",
+          "pattern": "^[0-9a-f]{8}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{12}$"
+        },
+        "studyUuid": {
+          "type": "string",
+          "pattern": "^[0-9a-f]{8}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{12}$"
+        },
+        "user_id": {
+          "type": "number"
+        },
+        "workflowUuid": {
+          "type": "string",
+          "pattern": "^[0-9a-f]{8}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{12}$"
+        }
+      }
+    }
+  ]
+}
```

Comment:
Unless I'm missing something, I think you really want something like:
```
required:
  custom_name
  studyUuid
  user_id
  workflowUuid
oneOf:
  toolUuid
  NodeSetUuid
  NodeRelationship
```
instead of copy-and-pasting the first four.